tracing-subscriber/src/macros.rs

```diff
@@ -0,0 +1,38 @@
+macro_rules! try_lock {
+    ($lock:expr) => {
+        try_lock!($lock, else return)
+    };
+    ($lock:expr, else $els:expr) => {
+        if let Ok(l) = $lock {
+            l
+        } else if std::thread::panicking() {
+            $els
+        } else {
+            panic!("lock poisoned")
+        }
+    };
+}
+
+/// Declares fmt items.
+#[doc(hidden)]
+macro_rules! cfg_fmt {
+    ($($item:item)*) => {
+        $(
+            #[cfg(feature = "fmt")]
+            #[cfg_attr(docsrs, doc(cfg(feature = "fmt")))]
+            $item
+        )*
+    }
+}
```

Comment:
You could make this one macro instead of two by taking the feature as a parameter, something like this:
```suggestion
macro_rules! cfg_fmt {
    (#[feature = $name: lit ] $($item:item)*) => {
        $(
            #[cfg(feature = $name)]
            #[cfg_attr(docsrs, doc(cfg(feature = $name)))]
            $item
        )*
    }
}
```

Reply:
I fixed it. 
9b4f0e3